package.json

```diff
@@ -31,7 +31,9 @@
     "prettier": "^2.3.0"
   },
   "dependencies": {
+    "@wordpress/api-fetch": "^4.0.0",
+    "@wordpress/components": "^14.1.1",
     "@wordpress/dom-ready": "^2.13.2",
-    "@wordpress/i18n": "^3.20.0"
```

Comment:
Looks like we inadvertently removed the i18n dependency.  Can you please add it back and take a pass at wrapping any new user-facing strings for translation?

I'm adding a few suggestions as pointers, but we should ideally wrap them all.

Reply:
I've gone through and wrapped as many strings as I could find in the `__( string, domain )` format. I'm pretty sure I've gotten them all, but let me know if I've missed any